tests/ImageSharp.Tests/TestUtilities/ImageProviders/BlankProvider.cs

```diff
@@ -6,25 +6,46 @@
 namespace ImageSharp.Tests
 {
     using System;
+    using Xunit.Abstractions;
 
     public abstract partial class TestImageProvider<TColor>
         where TColor : struct, IPixel<TColor>
     {
-        private class BlankProvider : TestImageProvider<TColor>
+        private class BlankProvider : TestImageProvider<TColor>, IXunitSerializable
         {
             public BlankProvider(int width, int height)
             {
                 this.Width = width;
                 this.Height = height;
             }
+            public BlankProvider()
+            {
+                this.Width = 100;
+                this.Height = 100;
+            }
 
             public override string SourceFileOrDescription => $"Blank{this.Width}x{this.Height}";
 
-            protected int Height { get; }
+            protected int Height { get; private set; }
 
-            protected int Width { get; }
+            protected int Width { get; private set; }
 
             public override Image<TColor> GetImage() => this.Factory.CreateImage(this.Width, this.Height);
+
+
+            public override void Deserialize(IXunitSerializationInfo info)
+            {
+                this.Width = info.GetValue<int>("width");
+                this.Height = info.GetValue<int>("height");
+                base.Deserialize(info);
+            }
+
+            public override void Serialize(IXunitSerializationInfo info)
+            {
+                info.AddValue("width", this.Width);
```

Comment:
Sometimes theory arguments work without being explicitly `IXunitSerializable`, sometimes they don't. 
Have you figured out the magic parts? :)